test/core/Bounds.js

```diff
@@ -1,19 +1,17 @@
 'use strict';
 
-describe('getBounds', function ()
+describe('getBounds', () =>
 {
-    it('should register correct width/height with a LOADED Sprite', function ()
+    it('should register correct width/height with a LOADED Sprite', () =>
```

Comment:
By converting all the tests to use fat arrows, we lose the Mocha context for doing things like `this.timeout(3000)`, `this.slow(500)` as well as passing object and functions between tests. While I like the `var` converting to `const`, I think the test callbacks should remain as functions.